project/TestGroups.scala

```diff
@@ -56,7 +61,27 @@ object TestGroups {
       "tests.MessagesSuite", "tests.pants.PantsGlobsSuite",
       "tests.SelectBspServerSuite", "tests.InverseDependenciesSuite",
       "tests.pantsbuild.PantsProjectNameSuite", "tests.TrigramSubstringsSuite",
-      "tests.TimerSuite", "tests.FoldingRangesSuite")
+      "tests.TimerSuite", "tests.FoldingRangesSuite",
+      "tests.rangeFormatting.MultilineStringRangeFormattingWhenPastingSuite",
+      "tests.codeactions.PatternMatchRefactorLspSuite",
+      "tests.ScalaToplevelLibrarySuite",
+      "tests.decorations.SyntheticHoverLspSuite",
+      "tests.rangeFormatting.MultilineStringRangeFormattingWhenSelectingSuite",
+      "tests.rangeFormatting.IndentWhenPastingSuite",
+      "tests.DebugDiscoverySuite", "tests.SemanticdbScala2Suite",
+      "tests.SelectionRangeLspSuite",
+      "tests.codeactions.OrganizeImportsLspSuite",
+      "tests.codeactions.InsertInferredTypeLspSuite",
+      "tests.FoldingRangeScala2Suite", "tests.SystemProcessSuite",
+      "tests.formatting.OnTypeFormattingSuite", "tests.DefinitionScala3Suite",
+      "tests.AnsiFilterSuite", "tests.DocumentSymbolScala3Suite",
+      "tests.ClassBreakpointSuite", "tests.FoldingRangeScala3Suite",
+      "tests.ScalafmtConfigSuite", "tests.DocumentSymbolScala2Suite",
+      "tests.StacktraceParseSuite", "tests.WorksheetDependencySourcesSuite",
+      "tests.DefinitionScala2Suite", "tests.AnalyzeStacktraceLspSuite",
+      "tests.codeactions.ExtractRenameMemberLspSuite",
+      "tests.debug.DotEnvFileParserSuite", "tests.SemanticdbScala3Suite",
```

Comment:
😆 totally didn't realize we were missing this many.

Reply:
It's been a while since I added it, so a lot of tests popped up :sweat_smile: 

It should be better with the warning.